src/Indices/Mapping.php

```diff
@@ -90,6 +91,13 @@ public function disableSource(): self
         return $this;
     }
 
+    public function dynamic(mixed $dynamic): self
```

Comment:
This package also supports PHP 7.4, so we better use annotations here:

```suggestion
    /**
     * @param string|bool $dynamic
     */
    public function dynamic($dynamic): self
```